airflow/example_dags/example_bigquery_to_bigquery.py

```diff
@@ -0,0 +1,34 @@
+import airflow
+from airflow.models import DAG
+from airflow.operators import bash_operator
+from airflow.contrib.operators.bigquery_to_bigquery import BigQueryToBigQueryOperator
+
+args = {
+    'owner': 'Airflow',
+    'start_date': airflow.utils.dates.days_ago(2),
+}
+
+dag = DAG(
+    dag_id='example_bigquery_to_bigquery',
+    default_args=args,
+    schedule_interval="@daily",
+)
+
+# [START howto_operator_bq_to_bq]
+copy_table = BigQueryToBigQueryOperator(
+    task_id='bq_to_bq_example',
```

Comment:
I think it's not a rule, however keeping `task_id` same as the name of task is a good practise. 
```suggestion
    task_id='copy_table',
```

Reply:
Thank you for your suggestions, I will accommodate all the changes and raise a new pull request.